src/send.ts

```diff
@@ -7,14 +7,13 @@ export type BodyType = 'json' | 'ndjson';
 
 export type Body = {
   body?: string;
-  json?: {
-    logs: unknown;
-  };
+  json?: Record<string, unknown> | Record<string, unknown>[];
```

Comment:
**suggestion:** make the `unknown` a generic.

Reply:
I improved types but I wasn't able to make ts compiler happy with generics, ...

Reply:
Hmmm interesting okay no worries. Maybe can improve that later.